src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -63,50 +63,50 @@ _* There are a few Nerdlets that basic users are allowed to use. See [Basic user
 
 ## Nerdpack permissions: Original user model
 
-Learn the differences between how basic users, full users, and admins can use and manage Nerdpacks in the [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/).
+Learn the differences between how basic users, full platform users, and admins can use and manage Nerdpacks in the [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/).
```

Comment:
Re-reading this, I think we should leave this as "full users". I don't think we called full users "full platform users" in the original user model.